### ISSUE TITLE
PDF print in blank

### ISSUE THREAD

**Issue by leotada** [AuburnSounds/printed#23]
I created a PDF using this library and printing from Chrome to the printer, print in blank.
Any help?
[output.pdf](https://github.com/AuburnSounds/printed/files/3699173/output.pdf)

**Comment by p0nce** [comment]
That PDF works fine in Firefox and Chrome here, probably some sort of font trickery not done well enough for all printers. This could also be #9 

Does the PDF shows up correctly in Acrobat?

**Comment by leotada** [comment]
I'm using Linux, in Evince app yes show correctly.

**Comment by p0nce** [comment]
Could be a bug in the Linux driver / printer, I have no way to know what happens.

**Comment by leotada** [comment]
Have you tried to print this PDF in Windows and worked?

**Comment by p0nce** [comment]
Hello, no but good idea I will try:
- [ ] try printing the given PDF

**Comment by p0nce** [comment]
Pass validation in https://www.pdf-online.com/osa/validate.aspx
Trying https://www.pdf-tools.com...

**Comment by p0nce** [comment]
Found something with https://www.pdf-online.com/osa/repair.aspx

```
3-Heights(TM) PDF repair tool, evaluation license valid until unbounded
Open file.
Analyze Objects.
Analyze Outlines.
Analyze Pages.
0x0041851A - W - Graphics operator q is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator cm is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator Q is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator q is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator cm is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator Q is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator q is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator cm is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator Q is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator q is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator cm is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator Q is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator q is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator cm is not allowed in text object.
    - File: output.pdf
0x0041851A - W - Graphics operator Q is not allowed in text object.
    - File: output.pdf
0x00418114 - W - The file is corrupt and needs to be repaired.
    - Page No.: 1
    - File: output.pdf
Close file.
```

**Comment by p0nce** [comment]
Hello @leotada , I think this is fixed in `printed` v1.0.0

**Comment by p0nce** [comment]
The fix also fixed Acrobat Reader complaining while "Printing to PDF" in Windows
